node-tests/addon-test.js

```diff
@@ -691,7 +691,7 @@ describe('ember-cli-babel', function() {
       expect(pluginRequired).to.be.true;
     });
 
-    it('returns false when targets do not require plugin', function() {
+    it.only('returns false when targets do not require plugin', function() {
```

Comment:
@existentialism not a problem. Just a surprise, glad it was easy to work around.